jenkins/job-configs/kubernetes-jenkins/kubernetes-garbagecollector.yaml

```diff
@@ -72,5 +72,115 @@
                 export KUBEMARK_NUM_NODES="100"
                 # The kubemark scripts build a Docker image
                 export JENKINS_ENABLE_DOCKER_IN_DOCKER="y"
+        - '100-gce': # kubernetes-garbagecollector-100-gce
```

Comment:
I agree - running it in all configurations seems excessive, I think that 2k should suffice. Moreover I don't really see the reason for running it daily. IIUC the only thing that can impact GC performance are changes in GC itself right? If so we can run this test weekly just in case, and manually with GC changes.

Having jenkins job set up has a lot of sense for sense even when they're not running continuously (currently both our real 2k node tests are like that), as it make running the test a matter of pushing a button.

Correctness tests should be run continuously of course.